facia-press/conf/routes

```diff
@@ -2,4 +2,7 @@ GET        /_healthcheck        conf.HealthCheck.healthcheck()
 GET        /                    controllers.Application.index
 GET        /debug/config        controllers.Application.showCurrentConfig
 
-GET        /pressed/live/*id      controllers.Application.generateLivePressedFor(id)
+GET        /pressed/live/*id    controllers.Application.generateLivePressedFor(id)
```

Comment:
That endpoint already existed to show what will be output for a given path. Should probably be `path`.